manifests/install.pp

```diff
@@ -18,7 +18,7 @@
   $python = $::python::version ? {
     'system' => 'python',
     'pypy'   => 'pypy',
-    default  => "${python::version}",
```

Comment:
I think this is here to ensure that python::version is a string, by casting it as a string. Not sure if it is actually needed though.